### ISSUE TITLE
player collision 

### ISSUE THREAD

**Issue by Rubie-be** [OldUnreal/UnrealTournamentPatches#849]
Playing coopgame map UB-Morose playerpawn get bigger collision after a while could not be ported up by mover second time
![gotblocked](https://user-images.githubusercontent.com/72510173/172013179-17425a1f-de2c-4c4d-84db-62b8fc1229d2.png)
?

**Comment by Rubie-be** [comment]
![image](https://user-images.githubusercontent.com/72510173/172013794-01c4aad1-22f2-42f0-a161-18591d5602b2.png)
projectiles do not fade out

**Comment by SeriousBarbie** [comment]
Map UB-Morose has no embedded code nor references to unusual U-files. Maybe the game controller or a mutator causes changes of collision size.
And btw: what mover do you refer to?
![MoversInUB-Morose unr](https://user-images.githubusercontent.com/81397053/172045561-2ef2d754-494a-4cbe-ae7e-e475fe22c69d.jpg)

**Comment by Rubie-be** [comment]
This is where  it happen , but it ain't the mover  as I can stand on it and it moves up and get blocked when player collison reach the begin from the square hole up then the mover return..

![image](https://user-images.githubusercontent.com/72510173/172118574-6ca68437-a366-420b-a6a6-bbe7a808ce58.png)

![image](https://user-images.githubusercontent.com/72510173/172118708-413870bb-d204-425d-acc0-760c033640be.png)

**Comment by Rubie-be** [comment]
I did used  the fly command and could not fly up neither something blocked me , so I suicide  :)

**Comment by Rubie-be** [comment]
the trigger has collison radius from 180  could this be the case ??? we talk about 469c

**Comment by SeriousBarbie** [comment]
No, that Trigger8 opens the upper cover (Mover68) so that players can pass the opening.

I have loaded the map into UnrealEd2.2 and tested it - no problems with passing that way.
Have you tried running the map without any mutators?

**Comment by Rubie-be** [comment]
I know all this and that map is fine no doubt about this , I did report as bug in 469C
